thirdparty/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 
+set(MESHLIB_THIRDPARTY_DIR "${PROJECT_SOURCE_DIR}")
```

Comment:
don't we have such in root CMakeLists? looks like it is override of existing variable

Reply:
Yes, there is such variable. But does `thirdparty/CMakeLists.txt` gets ever invoked by other makelists?
If it is used as root in a separate script, as in `build_thirdparty.sh`, there are no such variables and we need to define one.